src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -780,5 +781,25 @@ public async Task When_TextBox_Wrap_Fluent()
 			Assert.AreEqual(height1, height3);
 			height2.Should().BeGreaterThan(height1);
 		}
+
+		[TestMethod]
+		public async Task When_Paste()
+		{
+			var SUT = new TextBox();
+
+			var pasteRaised = false;
+			SUT.Paste += (_, _) => pasteRaised = true;
```

Comment:
It might be a better idea to have this as `pasteCount` instead and assert it's 1 so we are sure the event doesn't fire multiple timess.